srcpkgs/helden-software/patches/0002-fix-file-locations.patch

```diff
@@ -0,0 +1,33 @@
+commit 53d52967b979a304feb47ba1c0bd880e268964b9
+Author: Jan Christian Grünhage <jan.christian@gruenhage.xyz>
+Date:   Sat Sep 25 10:32:19 2021 +0200
+
+    fix file locations
+
+diff --git a/usr/games/helden-software b/usr/games/helden-software
+index 9105ab5..e3622cd 100755
+--- a/usr/games/helden-software
++++ b/usr/games/helden-software
+@@ -2,6 +2,6 @@
+ JavaArgs="-Xms40m -Xmx256m"
+ HeldenArgs=""
+ if [ -f /etc/default/helden-software ] ; then
+-    . /etc/default/helden-software
++    source /etc/default/helden-software
```

Comment:
Why this change?

Reply:
Good question, I don't remember :D

I should probably drop that change there

Reply:
Resolved, I've removed that part now.